server/models/dataset_helpers/image_metadata.py

```diff
@@ -513,6 +535,18 @@ def _checkMetadataErrors(clinical):
                 names=[DiagnosisFieldParser.name, BenignMalignantFieldParser.name],
                 values=[diagnosis, benignMalignant])
 
+    # Set melanocytic field based on diagnosis
+    if diagnosis in _melanocyticDiagnoses:
+        if not melanocytic:
```

Comment:
Since `melanocytic` could also be `None`, I think it's cleaner to explicitly check whether it's `True` or `False` here and below.

Reply:
The validation process has already populated the field with `True` or `False` at this point, right?

Reply:
Yes, but I think it's more robust to not have to depend on that assumption in later parts of the code.